src/python/pants/backend/rust/lint/rustfmt/skip_field.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from pants.backend.rust.target_types import RustCrateTarget
+from pants.engine.target import BoolField
+
+
+class SkipRustfmtField(BoolField):
+    alias = "skip_rustfmt"
+    default = False
+    help = "If true, don't run rustfmt on this crate."
+
+
+def rules():
+    return [RustCrateTarget.register_plugin_field(SkipRustfmtField)]
```

Comment:
When users run `pants fmt`, should the entire Rust package be formatted? Then this field should be placed on `RustPackageTarget` instead of `RustCrateTarget`.